addon/utils/errors.js

```diff
@@ -1,10 +1,8 @@
 import Ember from 'ember';
 
 export function getContext(router) {
-  var infos = router.currentState.routerJsState.handlerInfos;
-
   var url = router.get('location').getURL();
-  var routeName = infos[infos.length - 1].name;
+  var routeName = router.currentRoute.name;
```

Comment:
The router service was rolled out in 2.15 and should allow it to pass ember-try scenarios by checking the verion. Should probably add a scenario for 2.15? 
```suggestion
  var routeName;
  if(Ember.VERSION >= '2.15') {
    routeName = router.currentRouteName;
  } else {
    routeName = infos[infos.length - 1].name;
  }

Reply:
Good catch!